### DISCUSSION TITLE
JSON File

### DISCUSSION THREAD

**Discussion by DerekWongMI** [rrirower/DP2SceneGraphWizard#20] [General]
Can I have a sample JSON file to check against mine? Since I am getting a  ParseJSON: Data is empty: pkg:/components/tasks/MainLoaderTask.brs , I am suspecting my JSON file is not up to spec.

**Comment by ferdiworks** [comment]
If you post your json feed link we can take a look at it

**Reply by rrirower** [reply]
Or, use the Channel Feed Manager software to validate your feed file.

**Reply by DerekWongMI** [reply]
I got something workable thanks to you guys and the Channel Feed Manager. Thank you.
It didn't like my original category.

**Comment by rrirower** [comment]
Resolved using the [Channel Feed Manager](https://github.com/rrirower/Channel-Feed-Manager).
